src/DBMetadata/metadataRetriever.ts

```diff
@@ -1,9 +1,9 @@
 import DBMetadata from '../models/dbMetadata';
 import ConnData from '../models/connData';
 
-interface IMetadataRetriever {
+interface MetadataRetriever {
     getSchemaInfo(connString: string): Promise<DBMetadata[]>;
     buildConnectionString(info: ConnData): string;
 }
 
-export default IMetadataRetriever
\ No newline at end of file
+export default MetadataRetriever
```

Comment:
new line

Reply:
?

Reply:
There should always be a new line at the end of the a file

Reply:
Why?

Reply:
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

Reply:
I'll add it to the formatter so we won't have to worry about it ourselves, so you don' t need to fix it if you don't want to